docs/testrunner-toolkit/configuration/espresso.md

```diff
@@ -234,101 +443,143 @@ testOptions:
-
-__Description__: Instructs `saucectl` to only run the specified classes for this test suite.
+#### `class`
+<p><small>| OPTIONAL | ARRAY |</small></p>
 
-__Type__: *array*
+Instructs `saucectl` to only run the specified classes for this test suite.
 
-__Example__:
 ```yaml
   class:
     - com.example.android.testing.androidjunitrunnersample.CalculatorAddParameterizedTest
 ```
+---
 
-### `notClass`
-
-__Description__: Instructs `saucectl` to run all classes for the suite *except* those specified here.
+#### `notClass`
+<p><small>| OPTIONAL | ARRAY |</small></p>
 
-__Type__: *array*
+Instructs `saucectl` to run all classes for the suite *except* those specified here.
 
-__Example__:
 ```yaml
   notClass:
     - com.example.android.testing.androidjunitrunnersample.CalculatorInstrumentationTest
 ```
+---
 
-### `size`
-
-__Description__: Instructs `saucectl` to run only tests that are annotated with the matching size value.
-
-__Type__: *enum*
+#### `size`
+<p><small>| OPTIONAL | ENUM |</small></p>
 
-__Values__: *small*, *medium*, *large*
+Instructs `saucectl` to run only tests that are annotated with the matching size value. Valid values are `small`, `medium`, or `large`. You may only specify one value for this property.
 
-__Example__:
 ```yaml
   size: small
 ```
+---
 
-### `package`
-
-__Description__: Instructs `saucectl` to run only tests in the specified package.
+#### `package`
+<p><small>| OPTIONAL | STRING |</small></p>
 
-__Type__: *string*
+Instructs `saucectl` to run only tests in the specified package.
 
-__Example__:
 ```yaml
   package: com.example.android.testing.androidjunitrunnersample
 ```
+---
 
-### `annotation`
-
-__Description__: Instructs `saucectl` to run only tests that match a custom annotation that you have set.
+#### `annotation`
+<p><small>| OPTIONAL | STRING |</small></p>
 
-__Type__: *string*
+Instructs `saucectl` to run only tests that match a custom annotation that you have set.
 
-__Example__:
 ```yaml
   annotation: com.android.buzz.MyAnnotation
 ```
+---
 
-### `numShards`
+#### `numShards`
+<p><small>| OPTIONAL | INTEGER |</small></p>
 
-__Description__: Sets the number of separate shards to create. Read more about shard tests on the [Android developer site](https://developer.android.com/training/testing/junit-runner#sharding-tests).
+Sets the number of separate shards to create for the test suite. Read more about shard tests on the [Android developer site](https://developer.android.com/training/testing/junit-runner#sharding-tests).
 
-When `numShards` is greater than 1, `saucectl` will create the create the sharded jobs automatically.
+When sharding is configured, `saucectl` creates the sharded jobs automatically for each of the devices defined for the suite based on the number of shards you specify. For example, if a suite defines a single emulator version with 2 shards, `saucectl` clones the suite and runs one shard index on the first suite, and the other shard index on the identical clone suite.
```

Comment:
Should we specify that if there is multiple devices / emulators + shards everything gets multiplied ?
2 devices + 2 emulators + 2 shards results in 8 jobs being triggered ?